### PR TITLE
New term entry for .nextAfter()

### LINKED ISSUE #906: [Term Entry] Java Math Methods: `.nextAfter()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.nextAfter()` Java method.

- In the `terms/` folder, make a new folder called `next-after`
- Inside the `next-after` folder, create a new markdown file called `next-after.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by YanisaHS**
### Description

I added the `.nextAfter()` Java Math method in `java/concepts/math-methods/terms/`. The file:

- Defines `.nextAfter()`
- Provides the syntax
- Provides an example

Closes #906 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@YanisaHS, I did a first review and it looks good as is. So on to review number two.

File: content/java/concepts/math-methods/terms/next-after/next-after.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+Title: '.nextAfter()'
+Description: 'A method used to get the floating-point number next to the first argument in the direction of the second argument.'
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/next-after/next-after.md:3]:
```suggestion
Description: 'Returns the floating-point number next to the first argument in the direction of the second argument.'
```

File: content/java/concepts/math-methods/terms/next-after/next-after.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **Math.nextAfter()** method is used to get the floating-point number next to the first argument in the direction of the second argument.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/next-after/next-after.md:16]:
```suggestion
The **Math.nextAfter()** method returns the floating-point number next to the first argument in the direction of the second argument.
```

File: content/java/concepts/math-methods/terms/next-after/next-after.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The following example demonstrates using `.nextAfter()`:
+
+```java
+public class Test {
+    public static void main(String args[]) {
+        float start = 1.15f;
+        double direction = 5.37;
+        System.out.println(Math.nextAfter(start, direction));
+    }
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/next-after/next-after.md:44]: Can we do 2-space indents?

File: content/java/concepts/math-methods/terms/next-after/next-after.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+The **Math.nextAfter()** method is used to get the floating-point number next to the first argument in the direction of the second argument.
+
+## Syntax
+
+```pseudo
+Math.nextAfter(start, direction)
+```
+
+The first argument can be of type `double` or `float`, but the second argument can only be of type `double`. The return type for `.nextAfter()` is `double` or `float`, and matches the type of the first argument.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/next-after/next-after.md:24]:
```suggestion
The first argument `start` can be of type `double` or `float`, but the second argument `direction` can only be of type `double`. The return type for `.nextAfter()` is of `double` or `float`, and matches the type of the first argument.
```

File: content/java/concepts/math-methods/terms/next-after/next-after.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Some special cases for `.nextAfter()` include:
+
+- If one or both arguments are `NaN`, the result is `NaN`.
+- If both arguments are signed zero, the direction of the result is unchanged.
+- If the first argument is positive or negative `Double.MIN_VALUE`, and the second argument has a value which would return a result with a smaller magnitude, zero is returned with the same sign as the first argument.
+- If the first argument is infinity and the second argument has a value which would return a result with a smaller magnitude, `Double.MAX_VALUE` is returned with the same sign as the first argument.
+- If the first argument is positive or negative `Double.MAX_VALUE`, and the second argument has a value which would return a result with a larger magnitude, infinity is returned with the same sign as the first argument.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/next-after/next-after.md:32]: Can we refer to the content field values (e.g., `MAX_VALUE` ((2-2-52)·21023)) without calling it on the `Double` object?

**Review by yangc95 — COMMENTED** [review]
@YanisaHS I went ahead and did a second review!

Please let me know if you have any concerns with any of my comments and tag me once you've looked them over.

File: content/java/concepts/math-methods/terms/next-after/next-after.md (hunk 3 above)
Comment (YanisaHS) [reply content/java/concepts/math-methods/terms/next-after/next-after.md]: Done in a different commit!

File: content/java/concepts/math-methods/terms/next-after/next-after.md (hunk 5 above)
Comment (YanisaHS) [reply content/java/concepts/math-methods/terms/next-after/next-after.md]: Done in a different commit!

**Comment by YanisaHS** [comment]
Thank you @yangc95 !! 🥳  Done!

**Review by yangc95 — APPROVED** [review]

File: content/java/concepts/math-methods/terms/next-after/next-after.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- If the first argument is infinity and the second argument has a value which would return a result with a smaller magnitude, `MAX_VALUE` is returned with the same sign as the first argument.
+- If the first argument is positive or negative `MAX_VALUE`, and the second argument has a value which would return a result with a larger magnitude, infinity is returned with the same sign as the first argument.
+
+## Example
+
+The following example demonstrates using `.nextAfter()`:
+
+```java
+// Test.java
+
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/next-after/next-after.md:45]:
```suggestion
```

**Comment by yangc95** [comment]
@YanisaHS live link: https://www.codecademy.com/resources/docs/java/math-methods/next-after

cc: @SSwiniarski @Dusch4593

**Comment by yangc95** [comment]
@YanisaHS 😆 there's a missing period in the first bullet point which is bugging me a bit 😅 that was my bad -- but would you be able to fetch from `main` open a quick typo fix PR to add that change?

**Comment by YanisaHS** [comment]
@yangc95 😰 🤣  YES I will do that!

**Pull request merged**